src/core/ix86/iR3000A.cc

```diff
@@ -102,22 +107,20 @@ class X86DynaRecCPU final : public PCSX::R3000Acpu {
         if (m_delayedLoadInfo[other].index == index) m_delayedLoadInfo[other].active = false;
     }
 
-    PCSX::ix86 gen;
-
     uintptr_t *m_psxRecLUT;
-    static const size_t RECMEM_SIZE = 8 * 1024 * 1024;
+    static constexpr size_t RECMEM_SIZE = 8 * 1024 * 1024;
+    CodeGenerator gen;
 
-    int8_t *m_recMem; /* the recompiled blocks will be here */
-    char *m_recRAM;   /* and the s_ptr to the blocks here */
-    char *m_recROM;   /* and here */
+    int8_t *m_recMem;    /* the recompiled blocks will be here */
```

Comment:
It might be more judicious to replace it with void pointers? You wouldn't need any ugly cast anymore.

Reply:
Good point. However thanks to the new generator's nice memory management utilities, m_recMem can just... be removed because it did nothing useful anymore.
The other pointers to raw bytes (m_recRAM, m_recROM) were retyped to uint8_t* so the `char*` doesn't mislead anyone into thinking they're meant to be strings or anything of the sort.